sdk/appcenter-distribute/src/main/res/values/appcenter_distribute.xml

```diff
@@ -17,6 +17,7 @@
     <!-- TODO: Deprecated. Remove this key and move the translated strings to appcenter_distribute_downloading_update. -->
     <string name="appcenter_distribute_downloading_mandatory_update">Downloading update</string>
     <!-- TODO: Add translations. -->
```

Comment:
@AnastasiaKubova Please add separate task about translations, if it's missing at the moment